be/src/vec/core/block.cpp

```diff
@@ -577,7 +586,7 @@ MutableColumns Block::mutate_columns() {
     size_t num_columns = data.size();
     MutableColumns columns(num_columns);
     for (size_t i = 0; i < num_columns; ++i) {
-        columns[i] = data[i].column ? (*std::move(data[i].column)).assume_mutable()
+        columns[i] = data[i].column ? (*std::move(data[i].column)).mutate()
```

Comment:
here maybe cause mutable column is different column in block，can not change the mem in block cause result errror